README.rst

```diff
@@ -279,9 +279,14 @@ Command Line
 Test
 ----
 
-The tests are written with python ``unittest``, to run them, use pytest::
+The tests are written with python ``unittest``, to run them, use
+`pytest <https://pypi.org/project/pytest/>`_::
 
-    pytest test.py
+    pytest
+
+If you get a failure like ``unsupported locale setting`` you may need to add
+extra locales that the tests use. Refer to the steps used in the
+`build process <.github/workflows/build.yml>`_.
```

Comment:
```suggestion
`build process <.github/workflows/build.yml#L25-L29>`_.
```

Reply:
I decided not to do that because the lines could move, so folks would have to use their own judgement to find them.

Although, adding lines now makes it easier to find right now - and if they move later, it's just the same as what I had. So yes let's go with what you added!